package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "chalk": "^4.1.0",
-    "shelljs": "^0.8.4"
+    "shelljs": "^0.8.4",
+    "cypress": "^5.4.0"
```

Comment:
Any reason to add cypress as a dependency ? If this was accidental, it would avoid changes on the lock too, so you could revert this file and yarn.lock.